src/utils/getParents.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import classNames from 'classnames';
+import { NavLink, useSearchParams } from 'react-router-dom';
+import { Person } from '../types';
+
+type Props = {
+  parents?: Person;
```

Comment:
```suggestion
  parent: Person;
```
it cannot be undefined 
<img width="496" alt="image" src="https://github.com/mate-academy/react_people-table-advanced/assets/70220667/7c1e5893-5d3c-4ecb-ae6a-cf49b22742f4">

you render this component only when parent exists